tools/COMPILE-PROTOS.sh

```diff
@@ -79,6 +79,11 @@ func (c *RegistryClient) GrpcClient() rpcpb.RegistryClient {
 }
 END
 
+# patch the generated GAPIC to send Authorization tokens with insecure requests
+# (this allows the registry command line tools to test container builds)
+sed -i 's/return metadata.NewOutgoingContext(ctx, out)/insecure := os.Getenv("APG_REGISTRY_INSECURE")\ntoken := os.Getenv("APG_REGISTRY_TOKEN")\nif insecure == "1" \&\& token != "" \{ \nout["authorization"] = append(out["authorization"], "Bearer "+token) \n\}\nreturn metadata.NewOutgoingContext(ctx, out)/' gapic/doc.go
```

Comment:
If this stops working will CI fail? My concern is that the generated code may change in the future, and whoever happens to bump the version won't manually verify this still works.

Reply:
Good point. If this breaks compilation might fail but it's not guaranteed - or this could just silently not happen and stop working. I'll update #52 to note that this was done. Perhaps we should keep that open until we have a better solution?

Reply:
That seems like a reasonable approach 👍